cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/ProvisioningService.java

```diff
@@ -51,19 +54,24 @@
    */
   public void reloadProvisioners() {
     Map<String, Provisioner> provisioners = provisionerProvider.loadProvisioners();
+    Map<String, ProvisionerConfig> provisionerConfigs = provisionerConfigProvider.loadProvisionerConfigs();
```

Comment:
I think `loadProvisionerConfigs()` should take in a `Set<String>` of provisioner names. That way you know which config files to look for.